### DISCUSSION TITLE
negative value in wallet graph

### DISCUSSION THREAD

**Discussion by 0ferreira** [Chia-Network/chia-blockchain#9390] [Support]
![image](https://user-images.githubusercontent.com/94022118/143501126-9453b203-1fc6-49bf-b9cd-9baf550d07c2.png)
![image](https://user-images.githubusercontent.com/94022118/143501164-f350f7c5-7189-4521-94dc-cfe6c782e302.png)

![image](https://user-images.githubusercontent.com/94022118/143500533-53d5c380-32f3-426d-9cc7-351a45cfd43a.png)
![image](https://user-images.githubusercontent.com/94022118/143500582-addd7a58-8d06-48ff-8406-4fab279b6dae.png)

Can you help me?

I'm farming at XCH and Space and at the two pools my accounts went to negative values ​​twice.
What happened?

**Comment by shaneo257** [comment]
is a known issue, a git issue has been lodged https://github.com/Chia-Network/chia-blockchain/issues/8832
